### PR TITLE
magnum: add x509keycert store option

### PR DESCRIPTION
**Pull request by sidthe**
Introduce a new certificate store called x509keypair to magnum.

### CODE REVIEW

File: crowbar_framework/app/helpers/barclamp/magnum_helper.rb [hunk 1]
```diff
@@ -21,7 +21,8 @@ def cert_manager_types(selected)
       options_for_select(
         [
           ["Local", "local"],
-          ["Barbican", "barbican"]
+          ["Barbican", "barbican"],
+          ["X509keypair", "x509keypair"]
```
Comment (vuntz) [inline crowbar_framework/app/helpers/barclamp/magnum_helper.rb]: Asking out loud: is it possible to have a string that is slightly more understandable than X509keypair in the webui? :-)

**Comment by sidthe** [comment]
@vuntz fair. Let's label it inline with the documentation and have it as Magnum Database. Is that allright?

**Comment by vuntz** [comment]
> @vuntz fair. Let's label it inline with the documentation and have it as Magnum Database. Is that allright?

Actually, I was trying to find what label would be best and your proposal is quite good. Maybe "Magnum Database (x509keypair)" so that it's also explicit what this is for experts?

**Comment by sidthe** [comment]
> Actually, I was trying to find what label would be best and your proposal is quite good. Maybe "Magnum Database (x509keypair)" so that it's also explicit what this is for experts?

I don't have a preference. Googling "magnum database certificate" leads to this result http://docs.openstack.org/developer/magnum/userguide.html which specifically tells what magnum database option is.

shall I append the x509keypair to the name?

**Comment by vuntz** [comment]
> shall I append the x509keypair to the name?

I'd do it, I guess, yes. But I won't fight over it.

**Comment by mjura** [comment]
I would prefer to keep it simple, my proposal is to change it to just "Database" which is most accurate.

**Comment by sidthe** [comment]
@vuntz changed
@mjura simply Database would also work for me. @vuntz has the final word

**Pull request merged**